tutorials/physics/using_character_body_2d.rst

```diff
@@ -415,21 +384,21 @@ And the code for the Bullet:
     extends CharacterBody2D
 
     var speed = 750
-    var velocity = Vector2()
 
-    func start(pos, dir):
-        rotation = dir
-        position = pos
+    func start(_position, _direction):
```

Comment:
I'm not very familiar with GDScript naming conventions but I don't think method parameters are usually prefixed by `_`.

Reply:
It's common when you're shadowing existing parameters like `position`.